apps/etterlatte-beregning/src/main/kotlin/sanksjon/SanksjonService.kt

```diff
@@ -19,6 +20,24 @@ class SanksjonService(
         return sanksjonRepository.hentSanksjon(behandlingId)?.sortedBy { it.fom }
     }
 
+    suspend fun kopierSanksjon(behandlingId: UUID, brukerTokenInfo: BrukerTokenInfo) {
+        logger.info("Kopierer sanksjoner fra forrige behandling med behandlingID=$behandlingId")
+        val behandling =  behandlingKlient.hentBehandling(behandlingId, brukerTokenInfo)
+        val forrigeBehandlingId = behandlingKlient.hentSisteIverksatteBehandling(behandling.sak, brukerTokenInfo).id
+
+        val sanksjoner = sanksjonRepository.hentSanksjon(forrigeBehandlingId)
+        if (sanksjoner != null) {
+            sanksjoner.forEach {
+                sanksjonRepository.opprettSanksjon(
+                    behandlingId,
+                    it.sakId,
+                    brukerTokenInfo.ident(),
```

Comment:
Har ikke sjekket hvordan det er løst for andre kopieringer, men stusser litt over at vi overskriver kilde. Jeg tror det er interessant for oss å ta med oss informasjon om hvilke saksbehandler som opprinnelig opprettet sanksjonene.

Reply:
Enig med deg der! Det er nok greit å ha med seg. Skal få fikset det